tests/datasource/fluent/test_sql_datasources.py

```diff
@@ -104,7 +104,6 @@ def test_table_quoted_name_type_all_upper_case_normalizion_is_noop():
             assert str(table_asset.table_name) == table_name
             assert str(table_asset.table_name.casefold()) != table_name
             assert isinstance(table_asset.table_name, sqlalchemy.quoted_name)
-            assert table_asset.table_name.quote is True
```

Comment:
This is not a useful thing to check.
What matters is that it is an instance of `sqlalchemy.quoted_name` but it doesn't (and often shouldn't) be `quote is True` (unless the original string actually was quoted).